frontend/src/pages/node/AssociationsTable.vue

```diff
@@ -259,13 +261,14 @@ const {
     /** catch case where no association categories available */
     if (!props.node.association_counts.length)
       throw Error("No association info available");
-
+    console.log("Include Orthologs:", props.includeOrthologs.valueOf());
     /** get association data */
     const response = await getAssociations(
       props.node.id,
       props.category.id,
       start.value,
       perPage.value,
+      props.includeOrthologs.valueOf(),
```

Comment:
This should just be `props.includeOrthologs`? I'm not sure what `valueOf` is doing there.